zerver/lib/test_classes.py

```diff
@@ -2308,6 +2309,7 @@ def test_user(self) -> UserProfile:
     @override
     def setUp(self) -> None:
         super().setUp()
+        self.legacy_urls: list[str] = []
```

Comment:
Created this separately from self.url to avoid having to modify the several webhooks that use `self.url` directly.